.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 12.22.8
```

Comment:
what's .tool-versions, is it necessary to be included in repo?

Reply:
It's a common pattern for IDEs and version managers (like asdf) to detect the best version to use

Reply:
The version not being specified helped the bug not to be detected

Reply:
will this have any effect in cli runtime?

Reply:
it only affects tools like ASDF or IDEs